Garrett-Iannuzzi/BYOB
# Add data

#### What does this PR do?
- Adds the two base datasets, titles and metros
- Adds the first migration under the `db` folder
- Add seeded data to Postgres in the `seeds` folder 

## How to test this?
- We can test this data by cloning down the repo 
- Run Postico to see the data reflected in the tables  

## Issues 
- This PR will close #1 and #2 
- They have already been closed on the project board 

## Notes
- Next move will be to start adding end points

@Garrett-Iannuzzi
